x-pack/legacy/plugins/security/server/lib/authorization/check_saved_objects_privileges.test.ts

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { SpacesPlugin } from '../../../../spaces';
+import { OptionalPlugin } from '../../../../../server/lib/optional_plugin';
+import { checkSavedObjectsPrivilegesWithRequestFactory } from './check_saved_objects_privileges';
+
+test(`checkPrivileges.atSpace when spaces is enabled`, async () => {
+  const expectedResult = Symbol();
+  const spaceId = 'foo-space';
+  const mockCheckPrivileges = {
+    atSpace: jest.fn().mockReturnValue(expectedResult),
+  };
+  const mockCheckPrivilegesWithRequest = jest.fn().mockReturnValue(mockCheckPrivileges);
+
+  const mockSpaces = {
+    isEnabled: true,
+    getSpaceId: jest.fn(),
```

Comment:
nit: if we leave the unused spaces "methods" undefined, then it'll throw an explicit error if we become dependent on it. it's primarily only helpful if we're depending on a method with only side-effects or that we anticipate to return `undefined`.